lib/rubocop/cop/rails/blank.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cops checks for code that can be changed to `blank?`.
+      # Settings:
+      #   NilOrEmpty: true
+      #     Convert checks for `nil` or `empty?` to `blank?`
+      #   NotPresent: true
+      #     Convert usages of not `present?` to `blank?`
+      #   UnlessPresent: true
+      #     Convert usages of `unless` `present?` to `blank?`
+      #
+      # @example
+      #   # bad
```

Comment:
I'd have good and bad section for each option here. That would make it easier to understand what they actually do.